addons/web_editor/static/src/js/common/utils.js

```diff
@@ -125,18 +125,18 @@ function _computePxByRem(toRem) {
  * @param {string} value
  * @param {string} unitTo
  * @param {string} [cssProp] - the css property on which the unit applies
- * @param {jQuery} [$target] - the jQuery element on which that css property
+ * @param {jQuery} [target] - the jQuery element on which that css property
  *                             may change
  * @returns {number}
  */
-function _convertValueToUnit(value, unitTo, cssProp, $target) {
+function _convertValueToUnit(value, unitTo, cssProp, target) {
```

Comment:
Why do you need to change something in web_editor ? We shouldn t have any dependence to web_editor. We will remove it